building/tooling/analyzers/interface.md

```diff
@@ -10,6 +10,7 @@ All interactions with the Exercism website are handled automatically. Analyzers
   - A path to a directory containing the submitted file(s) (with a trailing slash).
   - A path to an output directory (with a trailing slash). This directory is writable.
 - The script must write an `analysis.json` file to the output directory.
+- The script must write a `tags.json` file to the output directory.
```

Comment:
Suggest changing `must` to `should` so as not to cause existing analyzers to be violating the spec.

Reply:
Updated